packages/app/src/systems/Core/components/WalletInfo.tsx

```diff
@@ -35,6 +37,15 @@ export function WalletInfo({ onClose }: WalletInfoProps) {
           <AssetItem key={coin.assetId} coin={coin} />
         </div>
       ))}
+      <div className="flex justify-center mt-4 text-sm">
+        <Link
+          isExternal
+          className="flex items-center gap-2"
+          href={`${BLOCK_EXPLORER_URL}/address/${wallet?.address}`}
```

Comment:
Maybe add already the providerUrl as query param.

`${BLOCK_EXPLORER_URL}/address/${wallet?.address}?provider_url=${VITE_FUEL_PROVIDER_URL}`

Reply:
✅